apps/docs/src/app/containers/component-docs/fabric/fabric.component.html

```diff
@@ -22,7 +22,47 @@ <h2>Dialog</h2>
 >
   {{ sampleContent2 }} {{ sampleContent3 }}
   <fab-dialog-footer>
-    <fab-default-button (onClick)="clickSave()" text="Save" contentStyle="margin-right: 10px;"></fab-default-button>
+    <fab-default-button (onClick)="clickSave()" text="Save"></fab-default-button>
     <fab-default-button (onClick)="toggleDialog()" text="Cancel"></fab-default-button>
   </fab-dialog-footer>
 </fab-dialog>
+
+<hr />
+
+<h2>Combo box</h2>
+<fab-combo-box contentStyle="width: 300px;" (onChange)="comboChange($event)">
+  <options>
+    <fab-combo-box-option optionKey="A" text="See option A"></fab-combo-box-option>
+    <fab-combo-box-option optionKey="B" text="See option B"></fab-combo-box-option>
+  </options>
+</fab-combo-box>
+{{ selectedComboBoxKey }}: {{ selectedComboBoxValue }}
+
+<hr />
+
+<h2>Directive Calendar</h2>
+<fab-calendar (onSelectDate)="onSelectDate($event)">
+  <fab-calendar-strings
+    key="key"
```

Comment:
Same as the other comment - why is `key` needed?

Reply:
It's not, removed.